trade-finance-manager-api/src/constants/tasks.js

```diff
@@ -66,18 +98,26 @@ const MIA = {
         isConditional: true,
       },
       {
+        id: AIN_AND_MIA_ID.GROUP_1.CREATE_OR_LINK_SALESFORCE,
```

Comment:
task ids here need to be removed, they're generated dynamically. Can be changed - depending on the deal data, some tasks will or will not be added, so the task IDs need to be dynamic. Group IDs don't need to be dynamic right now